fog/overseer/server/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+#![feature(proc_macro_hygiene, decl_macro)]
+
+pub mod config;
+pub mod server;
+pub mod service;
```

Comment:
I agree. However, I think the `pub` modifier (which i think == export?) is needed here because the integration tests in the `test/` need to access these modules. If I remove `pub` for any of these modules it doesn't compile.